### DISCUSSION TITLE
Pairing fails with `verify/attestation/...` 404

### DISCUSSION THREAD

**Discussion by asood123** [WalletConnect/walletconnect-monorepo#4204] [Developer Support] [answered]
Hi there - 

I'm building a demo to allow an Account Abstraction Wallet to be used with a dapp via WalletConnect. I copied the code from [react-wallet-v2](https://github.com/WalletConnect/web-examples/tree/main/advanced/wallets/react-wallet-v2).

When I call `web3Wallet.pair({uri})`  (with a valid `wc...` string from say Uniswap interface), it always fails with a 404 to `verify/attestation/...` call. Things I have tried:

* verifying the domain by publishing on a vercel link but still same error. Is there a way to skip the verify call on localhost?
* I thought maybe the dapp can require a verify call, so I even spun up a local dapp with a different walletconnect id and still same error. 

404 occurs even before it triggers `session-proposal` (i.e. never reaches `web3wallet.on('session_proposal', onSessionProposal)`). I must be missing something obvious. Any pointers to debug why `verify/attestation/` call is being placed (I thought it shouldn't be) and how to either skip it or avoid the 404 would be greatly appreciated 🙏 .

cc @thekidnamedkd, who's running into something similar.

**Comment by Avkubs** [comment]
Interesting

**Comment by Ahara22** [comment]
Hmmm!

**Comment by 0xsaymon** [comment]
Same here.
At `localhost` working well, but get 404 for `/attestation/` endpoint at deployed app, with verified domain. 
It always happens after `web3wallet.pair({ uri })` and before `session_proposal`.

**Comment by millionpixeldfi** [comment]
> Hi there -
> 
> I'm building a demo to allow an Account Abstraction Wallet to be used with a dapp via WalletConnect. I copied the code from [react-wallet-v2](https://github.com/WalletConnect/web-examples/tree/main/advanced/wallets/react-wallet-v2).
> 
> When I call `web3Wallet.pair({uri})` (with a valid `wc...` string from say Uniswap interface), it always fails with a 404 to `verify/attestation/...` call. Things I have tried:
> 
> * verifying the domain by publishing on a vercel link but still same error. Is there a way to skip the verify call on localhost?
> * I thought maybe the dapp can require a verify call, so I even spun up a local dapp with a different walletconnect id and still same error.
> 
> 404 occurs even before it triggers `session-proposal` (i.e. never reaches `web3wallet.on('session_proposal', onSessionProposal)`). I must be missing something obvious. Any pointers to debug why `verify/attestation/` call is being placed (I thought it shouldn't be) and how to either skip it or avoid the 404 would be greatly appreciated 🙏 .
> 
> cc @thekidnamedkd, who's running into something similar.

I have the same issue, can anybody help ?

**Comment by mahnunchik** [comment]
The same issue...

**Comment by alexbalda** [comment]
Facing the same issue. Any solution to this, please?

**Comment by glitch-txs** [comment, marked as answer]
moved to https://github.com/WalletConnect/walletconnect-monorepo/issues/4639 please follow up there
